File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/cpp/ex02_maior_numero.cpp
/*
    Autor: Kael Elodie Whitmore
    Data: Junho/2026
    Descrição: Lê dois números inteiros e informa qual é o maior ou se são iguais.
*/

#include <iostream>
using namespace std;

int main()
{
    int numero1, numero2;

    cout << "Digite o primeiro numero: ";
    cin >> numero1;

    cout << "Digite o segundo numero: ";
    cin >> numero2;

    if (numero1 > numero2)
    {
        cout << "O maior numero e: " << numero1 << endl;
    }
    else if (numero2 > numero1)
    {
        cout << "O maior numero e: " << numero2 << endl;
    }
    else
    {
        cout << "Os numeros sao iguais." << endl;
    }

    return 0;
}

File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/python/ex02_maior_numero.py

#   Autor: Matheus F. Mation
#   Data: Agosto/2026
#   Descrição: Lê dois números inteiros e informa qual é o maior ou se são iguais.
num = 0
num2 = 0

print("Digite o primeiro numero: ")
num = int(input("num: "))

print("Digite o segundo numero: ")
num2 = int(input("num2: "))

if num > num2:
    print(f"o maior numero é {num}")
elif num2 > num:
    print(f"o maior numero é {num2}")
else:
    print(f"os dois numeros são iguais a {num}")